src/screens/Landing/index.js

```diff
@@ -84,7 +84,7 @@ export default function Landing() {
 
     return {
       inspectionId: urlParams.get('i'),
-      vehicleTypeParam: urlParams.get('v') ?? 1,
+      vehicleTypeParam: urlParams.get('v'),
```

Comment:
Change it with `vehicleTypeParam: (clientParam === 'pd8') ? urlParams.get('v') : urlParams.get('v') ?? 1`

If client is not `CAT` and `v` param is not in path, then default vehicle should be `cuv`.